addons/l10n_sa_edi/tests/test_account_move.py

```diff
@@ -0,0 +1,47 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import logging
+
+from odoo.exceptions import UserError
+from odoo.tests import tagged
+
+from .common import TestSaEdiCommon
+
+_logger = logging.getLogger(__name__)
+
+
+@tagged('post_install_l10n', '-at_install', 'post_install')
+class TestAccountMove(TestSaEdiCommon):
```

Comment:
```suggestion
class TestSAEdiAccountMove(TestSaEdiCommon):
```

it's better to have more unique name, it lets us more easily select the right test classes